README.md

```diff
@@ -121,6 +121,7 @@ autohide=autohide
 | Jump to next line                                     | <kbd>Ctrl</kbd> + <kbd>N</kbd> / <kbd>↓</kbd> |
 | Quit                                                  | <kbd>Ctrl</kbd> + <kbd>C</kbd>                |
 | Clear screen                                          | <kbd>Ctrl</kbd> + <kbd>L</kbd>                |
+| Auto Complete                                         | <kbd>Shift</kbd> + <kbd>Tab</kbd>             |
```

Comment:
I think the description over here should be a little more elaborate. Autocompletes what?

Reply:
Autocomplete will be used for autocompleting streams and mentions. Can you suggest a better description?